autoload/neoterm/test/golang.vim

```diff
@@ -0,0 +1,22 @@
+function! neoterm#test#golang#run(scope)
+  let command = 'go test'
+
+  if a:scope == 'file'
```

Comment:
`go test` runs test per package. There can be specified single test file but it needs everything that test need (imported packages) also specified.

http://stackoverflow.com/questions/16935965/how-to-run-test-cases-in-a-specified-file